x/tokenfactory/keeper/genesis_test.go

```diff
@@ -31,20 +31,22 @@ func (suite *KeeperTestSuite) TestGenesis() {
 			},
 		},
 	}
+
 	app := suite.App
-	// remove module account to ensure initGenesis initializes it on its own
-	tokenfactoryModuleAccount := app.AccountKeeper.GetAccount(suite.Ctx, app.AccountKeeper.GetModuleAddress(types.ModuleName))
-	app.AccountKeeper.RemoveAccount(suite.Ctx, tokenfactoryModuleAccount)
-	suite.Ctx = app.BaseApp.NewContext(false, tmproto.Header{})
+	suite.Ctx = app.BaseApp.NewContext(true, tmproto.Header{})
```

Comment:
Does it make sense to move this inside the helper and call the helper from here?

Reply:
I tried that but it didn't work, i get this error `kv store with key KVStoreKey{0xc0005142c0, acc} has not been registered in stores`

Reply:
actually nvm, fixed!